### ISSUE TITLE
Cannot get storedFile

### ISSUE THREAD

**Issue by pell-j** [l3nz/ari4java#131]
```      
      ari.channels().record(channel.getId(), "temp", "wav", 10, 1, "overwrite", true, "#")
```

temp.wav does exist in /var/spool/asterisk/recording

```
	System.out.println(ari.recordings().listStored());
		
	StoredRecording recording = ari.recordings().getStored("temp.wav");
		
	ari.recordings().getStoredFile("temp.wav", null);
```
list printed out below:

`      [ch.loway.oss.ari4java.generated.ari_4_0_0.models.StoredRecording_impl_ari_4_0_0@42b836]`

Error: (Some of the characters i could not copy and paste as they where not ascii or UTF-8. Not sure what is allow and not allow, I couldn't even paste it from the console to a text file.)
```
com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'RIFF¤': was expecting ('true', 'false' or 'null')
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1804)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:679)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._reportInvalidToken(ReaderBasedJsonParser.java:2835)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._handleOddValue(ReaderBasedJsonParser.java:1889)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:747)
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4141)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4000)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3004)
	at ch.loway.oss.ari4java.tools.BaseAriAction.deserializeJson(BaseAriAction.java:157)
	at ch.loway.oss.ari4java.generated.ari_4_0_0.actions.ActionRecordings_impl_ari_4_0_0.getStoredFile(ActionRecordings_impl_ari_4_0_0.java:323)
	at com.wickedstacks.pbx.HelloWorld.main(HelloWorld.java:71)

	at ch.loway.oss.ari4java.tools.BaseAriAction.deserializeJson(BaseAriAction.java:160)
	at ch.loway.oss.ari4java.generated.ari_4_0_0.actions.ActionRecordings_impl_ari_4_0_0.getStoredFile(ActionRecordings_impl_ari_4_0_0.java:323)
	at com.wickedstacks.pbx.HelloWorld.main(HelloWorld.java:71)

Caused by: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'RIFF
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1804)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:679)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._reportInvalidToken(ReaderBasedJsonParser.java:2835)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._handleOddValue(ReaderBasedJsonParser.java:1889)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:747)
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4141)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4000)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3004)
	at ch.loway.oss.ari4java.tools.BaseAriAction.deserializeJson(BaseAriAction.java:157)
	... 2 more


```

**Comment by pell-j** [comment]
`ari.recordings().getStoredFile("temp.wav");`

This line causes error. Emit null from above

**Comment by pell-j** [comment]
I see this in the asterisk console:

`[2019-12-08 22:10:17] VERBOSE[21399][C-00000066] app.c: x=0, open writing: /var/spool/asterisk/recording/temp format: wav, 0x744c53c8`

**Comment by pell-j** [comment]
similar problem:

https://github.com/asterisk/node-ari-client/issues/62

**Comment by pell-j** [comment]
duplicate #50

**Comment by grahambrown11** [comment]
Fixed in 0.7.0
